napari/layers/intensity_mixin.py

```diff
@@ -118,7 +125,6 @@ def contrast_limits_range(self, value):
             new_min = min(max(value[0], cur_min), value[1])
             new_max = max(min(value[1], cur_max), value[0])
             self.contrast_limits = (new_min, new_max)
-            self.events.contrast_limits()
```

Comment:
This was a duplicated event, since the `self.contrast_limits` setter already fires it.